### DISCUSSION TITLE
Sound & Push Notifications on Mobile App

### DISCUSSION THREAD

**Discussion by Stephen-Lobo** [chatwoot/chatwoot#2771] [Feature Requests]
When can we expect Sound and Push notifications on **Mobile** App ?

The Mobile App is wonderful, but it is missing Sound / Push notification for **new Message** on Android.

Hope you guys prioritize this very important feature.

**Comment by sojan-official** [comment]
We have this under work : https://github.com/chatwoot/chatwoot/pull/2757

**Reply by Stephen-Lobo** [reply]
Thank you for the update and great work.
